modules/recon/credentials-credentials/hashes_org.py

```diff
@@ -15,18 +15,17 @@ class Module(BaseModule):
             'Hash types supported: MD5, MD4, NTLM, LM, DOUBLEMD5, TRIPLEMD5, MD5SHA1, SHA1, MYSQL5, SHA1MD5, DOUBLESHA1, RIPEMD160',
             'Hashes.org is a free service. Please consider a small donation to keep the service running. Thanks. - @s3inlc'
         ),
-        'query': 'SELECT DISTINCT hash FROM credentials WHERE hash IS NOT NULL AND password IS NULL AND type IS NOT \'Adobe\'',
+        'query': 'SELECT DISTINCT hash FROM credentials WHERE hash IS NOT NULL AND password IS NULL AND type IS '
```

Comment:
This one is still wrapped, but that's ok. Let's go with the PEP8 version you submitted previously.

Reply:
This can be pushed to master now. Lets just leave it all for now and we can circle back around on the PEP8 stuff since it doesn't change it from working or not and we can do it for all modules.